code/modules/spells/spell_types/shapeshift.dm

```diff
@@ -14,10 +14,10 @@
 	var/shapeshift_type
 	var/list/current_shapes = list()
 	var/list/current_casters = list()
-	var/list/possible_shapes = list(/mob/living/simple_animal/mouse,\
-		/mob/living/simple_animal/pet/dog/corgi,\
-		/mob/living/simple_animal/hostile/carp/ranged/chaos,\
-		/mob/living/simple_animal/bot/ed209,\
+	var/list/possible_shapes = list(/mob/living/simple_animal/mouse,
+		/mob/living/simple_animal/pet/dog/corgi,
+		/mob/living/simple_animal/hostile/carp/ranged/chaos,
+//		/mob/living/simple_animal/bot/ed209,
```

Comment:
This bit I am soso on, On one hand, removes wizard ED's, on the other hand. Removes wizard ED's. Although to be honest, shapeshift is more or less a memespell. No real big loss.